include/rfb/rfbproto.h

```diff
@@ -1557,7 +1557,7 @@ typedef union {
 extern int rfbEncryptAndStorePasswd(char *passwd, char *fname);
 extern char *rfbDecryptPasswdFromFile(char *fname);
 extern void rfbRandomBytes(unsigned char *bytes);
-extern void rfbEncryptBytes(unsigned char *bytes, char *passwd);
+extern int rfbEncryptBytes(unsigned char *bytes, char *passwd);
```

Comment:
API change but I think it's OK ABI-wise. https://stackoverflow.com/questions/15626579/c-abi-is-changing-a-void-function-to-return-an-int-a-breaking-change also indicates so.

Reply:
What's the semantics of the return value? If it's boolean why not use the rfbBool type?